builtin/locale/__builtin.ru.tr

```diff
@@ -0,0 +1,246 @@
+Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Очистка всех объектов. Это может занять много времени. У вас может возникнуть тайм-аут. (запущено: @1)
+Cleared all objects.=Все объекты очищены.
+<name> <message>=<имя> <сообщение>
+Send a direct message to a player=Отправить прямое сообщение игроку
+Invalid usage, see /help msg.=Недопустимое использование, см. /help msg.
+The player @1 is not online.=Игрок @1 не находится в игре.
+DM from @1: @2=DM от @1: @2
+Message sent.=Сообщение отправлено.
+Get the last login time of a player or yourself=Вывести время последнего входа игрока или своё
+@1's last login time was @2.=Время последнего входа @1: @2.
+@1's last login time is unknown.=Время последнего входа @1 неизвестно.
+Clear the inventory of yourself or another player=Очистить свой инвентарь или инвентарь другого игрока
+You don't have permission to clear another player's inventory (missing privilege: @1).=У вас нет разрешения на очистку инвентаря другого игрока (отсутствует привилегия: @1).
+@1 cleared your inventory.=@1 очистил ваш инвентарь.
+Cleared @1's inventory.=Инвентарь @1 очищен.
+Player must be online to clear inventory!=Игрок должен быть онлайн, чтобы очистить его инвентарь!
+Players can't be killed, damage has been disabled.=Игроки не могут быть убиты, урон отключен.
+Player @1 is not online.=Игрок @1 не находится в игре.
+You are already dead.=Ты уже мертв.
+@1 is already dead.=@1 уже мертв.
+@1 has been killed.=@1 был убит.
+Kill player or yourself=Убить игрока или себя
+Invalid parameters (see /help @1).=Недопустимые параметры (см. /help @1).
+Too many arguments, try using just /help <command>=Слишком много аргументов, попробуйте использовать просто /help <команда>
+Available commands: @1=Доступные команды: @1
+Use '/help <cmd>' to get more information, or '/help all' to list everything.=Используйте '/help <cmd>', чтобы получить дополнительную информацию, или '/help all', чтобы перечислить все.
+Available commands:=Доступные команды:
+Command not available: @1=Команда недоступна: @1
+[all | privs | <cmd>] [-t]=[all | privs | <команда>] [-t]
+Get help for commands or list privileges (-t: output in chat)=Получить справку по командам или списку привилегий (-t: вывод в чате)
+Available privileges:=Доступные привилегии:
+Command=Команда
+Parameters=Параметры
+For more information, click on any entry in the list.=Для получения дополнительной информации нажмите на любую запись в списке.
+Double-click to copy the entry to the chat history.=Дважды щелкните, чтобы скопировать запись в историю чата.
+Command: @1 @2=Команда: @1 @2
+Available commands: (see also: /help <cmd>)=Доступные команды: (смотрите также: /help <команда>)
+Close=Закрыть
+Privilege=Привилегия
+Description=Описание
+print [<filter>] | dump [<filter>] | save [<format> [<filter>]] | reset=print [<filter>] | dump [<filter>] | save [<format> [<filter>]] | reset
```

Comment:
```suggestion
print [<filter>] | dump [<filter>] | save [<format> [<filter>]] | reset=print [<запрос>] | dump [<запрос>] | save [<формат> [<запрос>]] | reset
```

Reply:
I think `<filter>` -- `<фильтр>`. But I think it's not needed to translate, because people, who use this command know english. And also this command not available by default and not visible.

Reply:
> I think `<filter>` -- `<фильтр>`. But I think it's not needed to translate, because people, who use this command know english. And also this command not available by default and not visible. It's better just leave as is.

Hm, yes, profiler used rare and by experienced users who cares about server optimization. So yes, let's keep english variant